docs/environments/kind.md

```diff
@@ -0,0 +1,73 @@
+# `kind` - `k`ubernetes `in` `d`ocker
+
+[kind](https://github.com/kubernetes-sigs/kind) provides the quickest way to
+set up clusters for use with the Federation v2 control plane.
+
+## Download and Install kind
+
+If you don't yet have `kind` installed, you can run the following script to
+download and install a known working version.
+
+```bash
+./scripts/download-e2e-binaries.sh
+```
+
+Make sure that your `GOBIN` directory is in your path as that is where `kind`
+will be installed. Your `GOBIN` directory should be at `$(go env GOPATH)/bin`:
+
+## Create Clusters
+
+You can proceed to create clusters once you have `kind` available in your path.
+The `NUM_CLUSTERS` is `2` by default. Set that variable before invoking the
+script if you'd like to change the default.
+
+### Configure Insecure Container Registry
+
+Please answer the following questions to determine if you need to set up an
+insecure container registry on your host:
+
+1. Is this the first time you're running the `create-clusters.sh` script?
+2. Are you planning on creating container images locally without pushing to a
+public container registry such as quay.io?
+
+If you answered yes to both of these questions, then you will need to
```

Comment:
How about other cases?

- 1 No 2 Yes
- 1 No 2 No
- 1 Yes 2 No

Reply:
Is this really accurate? The teardown script also stops the local registry, iirc